lib/dalziel.rb

```diff
@@ -147,7 +147,7 @@ def does_not_match?(response)
       end
 
       def failure_message
-        if @request == []
+        if @request.blank?
```

Comment:
Blank is not useful, because that would make dalziel dependent on Rails/ActiveSupport

We should be able to just check for nil as well as emptyness

``` suggestion
        if @request.nil? || @request.empty?
```

Reply:
Oh right, I forgot `blank?` is rails, not ruby